### ISSUE TITLE
Make a report for showing which region nodes are along side non-region nodes

### ISSUE THREAD

**Issue by brendanheywood** [theCrag/website#422]
so we can fix em!

**Comment by scd** [comment]
@brendanheywood is this something we can get Campbell to follow up on, or do you want to do it. The fallback will be that I will do it, which is fine. I just want to get your thoughts so it is clear how we are going to finish this issue off.

**Comment by brendanheywood** [comment]
If we can get the report to @cgome is some easy fashion then that sounds great. That's one down side of the sql report method as he won't be able to re-run it at will.

**Comment by scd** [comment]
we could put it on our daily monitoring though

**Comment by brendanheywood** [comment]
That could work, although that email is pretty bloated, it would be great if it had a top section with was exceptions that needed action and then all the cruft if you want it.

**Comment by scd** [comment]
That is exactly how it works. I have defined dozens of exceptions, and if any of them trigger then it will appear at the top of the list. I never look through this other then the first line summary on gmail. For example it told us three months when the static server disk space was getting low.

**Comment by brendanheywood** [comment]
Ah cool, well that's a good thing then as we don't get many exceptions :)  I guess all region ~ crag siblings will be exceptions as we want strictly none of these?

**Comment by scd** [comment]
Yes, please see latest monitoring email, which links an exception to a non-empty report associated with the sql query file. The sql is run on production in this instance, but could be any server.

Closing issue now as we can manage the tidy up as part of our daily monitoring routine. Campbell also has access to this.

@cgome do you get the daily monitoring email. Can you start fixing regions with non-region siblings.

**Comment by cgome** [comment]
Yes. I'll be able to do this. Might need a bit of reminding/handholding to
get me started. Maybe when I'm back from Syd?

Campbell

**Comment by brendanheywood** [comment]
I've been working through this already, they fall into 4 camps:

* places where almost everything should be a region but isn't => make the odd ones out into a region
* places where almost everything should be a crag or other non-region => make the odd ones into crags
* places where the structure needs changing, often the case with emerging countries. I've generally left these alone as I need to know the locations of all the crags to see if there is a logical index structure etc. Needs consulting with a local, or more investigation
* places that legitimately are fine - this camp I didn't think would exist but I've found a couple that I just don't see much wrong with. In some cases I've made the 'offending' region into an area which contains crags. But this feels a little wrong to me. I think if we find cases like these we should add them here to discuss. examples below:

Example:
Should Bulahdelah be a crag or a region?
http://www.thecrag.com/climbing/australia/new-south-wales-and-act/hunter-valley

**Comment by brendanheywood** [comment]
Just realised this issue is close which makes it a little hard to find. Another example is where something should be a region, but if you change it you lose all the beta fields which happen to be filled out. eg:

http://www.thecrag.com/climbing/germany/sachsische-schweiz

See also #1807
